general-superstaq/general_superstaq/check/configs.py

```diff
@@ -79,12 +85,27 @@ def run(
     return 0
 
 
+def _trim_lines(lines: List[str], start_match: str) -> Tuple[List[str], int]:
+    """Remove package-specific configuration text, and identify the starting lines to compare."""
+    start_line = 0
+    if start_match in lines:
+        start_line = lines.index(start_match) + 2
+    return lines[start_line:], start_line
```

Comment:
nit (avoids iterating through the file twice)
```suggestion
    for start_line, line in enumerate(lines, start=2):
        if start_match in line:
            return lines[start_line:], start_line
    return lines, 0
```

also why the offset by 2? couldn't we just start comparing at the start_match line?

Reply:
The +2 is because we don't want to match the `start_match` (which isn't required to be present), we want to match what's below it... so +1... and then another +1 for an empty newline between `start_match` and the content to match (which looks cleaner because the content itself is broken into blocks separated by newlines -- this way it doesn't look like `start_match` is only associated with the first block).

Reply:
On second thought, it's simpler to just start at the `start_line` as in your example